src/components/dashboard/sidebar/styles/_SidebarCard.scss

```diff
@@ -0,0 +1,10 @@
+.card {
+    &.catalog-access-card {
+        border-top-width: 4px;
+        > .card-body {
+            > .card-title {
+                @extend .h4 !optional;
```

Comment:
I don't fully understand what `!optional` is doing here. Do you mind clarifying a bit? I haven't seen `!optional` used before. Does the .h4 styling only apply with `!optional`?

IIRC, this had to do with react-bootstrap always putting the `h5` class on the `.card-title`, so we're overriding it here?

Reply:
we discussed this offline but for reference: `!optional` isn't needed now that this isn't directly imported into SidebarCard.jsx